mongo.go

```diff
@@ -260,9 +260,12 @@ func (m Handler) Find(ctx context.Context, q *query.Query) (*resource.ItemList,
 	// If the number of returned elements is lower than requested limit, or not
```

Comment:
Could you please fix the existing comment typo? (not -> no).

Reply:
Done, added one comment from myself. Please review.